changelog.d/20240614_162207_regis_patch_dockerfile_common.md

```diff
@@ -0,0 +1 @@
+- [Feature] Introduce a "mfe-dockerfile-base" patch to customise the base layer of the "mfe" Docker image. (by @regisb)
```

Comment:
We should add similar info to README.rst section "Template patch catalog".

Reply:
Good point, I had forgotten that patches were documented in the readme.